packit/utils/repo.py

```diff
@@ -28,6 +28,7 @@ def get_repo(url: str, directory: Union[Path, str] = None) -> git.Repo:
     """
     directory = str(directory) if directory else tempfile.mkdtemp()
 
+    logger.info(f"Cloning upstream repository: {url}")
```

Comment:
Can you please change the `logger.debug` at line 34 to `info` instead so it's not printed when we don't clone anything?

https://github.com/packit/packit/blob/5f7e0b01123248aed8e82ba0187935e863f61f5f/packit/utils/repo.py#L117

Reply:
Ha, I still based my PR on `master` instead of `main`.

Reply:
Good point that when working with a local directory than it will be fast. OK, thanks for suggestion.